### PR TITLE
Latest adjustments to background decay0 v3.6 model

### PR DESCRIPTION
**Pull request by lpaulucc**
Clean up of old files + same configuration as Juergen's original files for decay0 v3.6.

### CODE REVIEW

File: dunesim/EventGenerator/Radiological/dune_radiological_model_decay0_hd_1x2x2.fcl [hunk 1]
```diff
@@ -14,7 +14,7 @@ BEGIN_PROLOG
 dune10kt_1x2x2_gammas_in_APAboards:  # 3.81 cm thick (2*4*4.76 mm), 16.5 cm in the vertical direction, and as wide as the APA frames on top of upper APAs and bottom of lower APAs - J. Reichenbacher (12/11/2023)
 {
    module_type:           "RadioGen"
-   PathToFile:             "Flux/Radionuclides/v03_06/"
+   PathToFile:             "Flux/Radionuclides/v03_05/"
```
Comment (absolution1) [inline dunesim/EventGenerator/Radiological/dune_radiological_model_decay0_hd_1x2x2.fcl:17]: Just double checking, is this downgrade here correct?

**Review by absolution1 — COMMENTED** [review]
Hi @lpaulucc 
PR looks good, I had one clarification question

File: dunesim/EventGenerator/Radiological/dune_radiological_model_decay0_hd_1x2x2.fcl (hunk 1 above)
Comment (lpaulucc) [reply dunesim/EventGenerator/Radiological/dune_radiological_model_decay0_hd_1x2x2.fcl:17]: We may want to change it in the future but turns out that when the flux file is the same as a previous version, they are not being copied to the new version folder, we are using what is already there.

Comment (absolution1) [reply dunesim/EventGenerator/Radiological/dune_radiological_model_decay0_hd_1x2x2.fcl:17]: Got it, thanks for clarifying

**Review by absolution1 — APPROVED** [review]

**Comment by lpaulucc** [comment]
trigger build

**Pull request merged**